template/express-rest-api/.gantry/dev.yml

```diff
@@ -1,8 +1,8 @@
-deployment: dev
+deployment: <%- devGantryEnvironmentName %>
 env: development
 
-maxInstanceCount: 1
 minInstanceCount: 1
+maxInstanceCount: 1
```

Comment:
Swap these two values around. It makes more sense, to me; to read `min` first and then `max`.
Also it caught a team out during a production incident 
https://github.com/SEEK-Jobs/hirer-access-api/commit/b45f858182d86dbde3a1315f5db5edadb59c719f